docs/optimization_models.rst

```diff
@@ -1,5 +1,610 @@
+The optimisation methods of this library are tools to assist the
+planning process of DHS projects and to analyze the economic feasibility of DHS
+for a given district, community or city - either by focusing on the DHS itself,
+or by also considering the overall energy system of a district, which could not
+just be the heating sector, but also the electricity, mobility sector or the
+gas infrastructure.
+
+At the moment, there is one approach using *oemof-solph* as linear optimisation library
+implemented. This approach is explained in the following sections. It totally makes sense to have
+some experiences with *oemof-solph* to understand this toolbox more easily.
+
+Scope
+-----
+
+The following questions can be addressed using the `optimize_investment` method
+of the *ThermalNetwork*:
+
+* What is the cost-optimal topology and dimensioning of a DHS piping system, 
+  given the locations of potential central heat supply plants, the potential
+  locations for the DHS piping system (e.g. street network),
+  and the position of consumers?
+* In addition to the first question, what is the cost-optimal expansion
+  of a given DHS system?
+* Is it cost-efficient to build a DHS at all, if there a consumer-wise heat
+  supply alternatives? (Comparison of central and de-central supply strategies)
+* What is the optimal dispatch of the heat producers? (In case there are no
+  expansion options, but just existing DHS pipes)
+* Planned: Streets-wise aggregation option
+
+To answer these questions, at the moment,
+the LP and MILP optimisation library oemof.solph is used.
+Other approaches, e.g. heuristic approaches, might follow.
+
+The following sections will give an overview about the general usage/workflow,
+(the necessary input data, the different optimisation settings and options,
+the results), and second, the underlying mathematical description.
+
+Usage
+-----
+
+Links to the subsections:
+
+* :ref:`Overview`
+* :ref:`Input Data`
+* :ref:`Label systematic <Label system>`
+* :ref:`Results`
+
+.. _Overview:
+
+Overview
+~~~~~~~~
+
+The optimisation of a given *ThermalNetwork* is executed by:
+
+.. code-block:: python
+
+    import dhnx
+
+    tnw = dhnx.network.ThermalNetwork()
+    
+    tnw = network.from_csv_folder('path/to/thermal_network')
+
+    invest_opt = dhnx.input_output.load_invest_options('path/to/invest_options')
+
+    tnw.optimize_investment(invest_options=invest_opt)
+
+For executing an optimisation, you must provide the `invest_options` additional to the previous
+data, which defines a *ThermalNetwork*. Both are explained in the following section.
+
+.. _Input Data:
+
+Input Data
+~~~~~~~~~~
+
+In this section, it is firstly revised, what input data is exactly necessary
+from the *ThemalNetwork* class, and then explained, what data needs to be
+provided as `invest_options` and as `settings`.
+
+The following figure provides an overview of the input data:
+
+.. 	figure:: _static/DHNx_Input_data.svg
```

Comment:
Even if I like the picture very much, it seems that some information does not apply in general or repeats specifications of ThermalNetwork. For the moment, I would be ok to leave it like it is to proceed with the release - maybe we can address this later?

Reply:
yeah, I already did some small updates in order to bring it more into line with the current state, however, as you say, there are still some issues. e.g. the sequences are also part of the thermal network. Other sequences are not ... I will update a more simplified version to give a rough overview.